support/cas-server-support-actions/src/main/java/org/apereo/cas/web/flow/TerminateSessionAction.java

```diff
@@ -113,4 +125,21 @@ private static boolean isLogoutRequestConfirmed(final RequestContext requestCont
         final HttpServletRequest request = WebUtils.getHttpServletRequestFromExternalWebflowContext(requestContext);
         return request.getParameterMap().containsKey("LogoutRequestConfirmed");
     }
+
+    /**
```

Comment:
Javadocs for getters/setters are generally not needed.

Reply:
I think I got an error from CheckStyle; I don't normally put JavaDoc on setters/getters. But I'll remove it...